webroot/plots/plotly/io500-mdtest-easy-write.html

```diff
@@ -15,9 +15,9 @@
+<script type="application/htmlwidget-sizing" data-for="htmlwidget-f967f755c2dc422fea13">{"viewer":{"width":1000,"height":300,"padding":"0","fill":false},"browser":{"width":1000,"height":300,"padding":"0","fill":false}}</script>
```

Comment:
I'm guessing that these lines are automatically generated by some program/tool?  It would be useful to describe how they are generated in the commit message.

Reply:
Not sure I'll be able to do that, as the code is automatically generated, please refer to:

https://github.com/IO500/webpage/blob/main/webroot/plots/plots.org